### ISSUE TITLE
loading this obj model crashed?

### ISSUE THREAD

**Issue by scolar** [viromedia/virocore#174]
the crash log as follow:
[log2.txt](https://github.com/viromedia/virocore/files/2247811/log2.txt)

the model file as follow:
[M076900K20102021903D_OBJ.zip](https://github.com/viromedia/virocore/files/2247814/M076900K20102021903D_OBJ.zip)

the .mtl file and .obj file contains chinese character,is it the reason which caused the crash?

**Comment by dthian** [comment]
Hey @scolar,

From [this](https://stackoverflow.com/questions/12127817/android-ics-4-0-ndk-newstringutf-is-crashing-down-the-app), it looks like you are right - the reason for the crash is most likely due to the Chinese characters. As a mitigation, is it possible to use english characters in your models?

**Comment by scolar** [comment]
At present,i will use english characters in my models,is the virocore can support chinese character names in 
future version? Thanks.

**Comment by dthian** [comment]
Yup, this will be fixed in future releases (not the upcoming one next week as the code for that is already locked down). I'll update this issue once the fixed is pushed to release.
